src/Microsoft.DotNet.Build.Tasks/PackageFiles/RepoAPI.Mapping.props

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Provides a default API implementation where possible by setting BuildTools properties/items.
+
+    API documentation: https://github.com/dotnet/source-build/tree/dev/release/2.0/Documentation/auto-dependency-flow
+  -->
+
+  <!-- Produce packages into the specified blob feed. -->
+  <PropertyGroup Condition="'$(DotNetOutputBlobFeedDir)' != ''">
+    <PackageOutputPath>$(DotNetOutputBlobFeedDir)packages/</PackageOutputPath>
```

Comment:
Do we think this is going to be setup in the correct order?

Sorry for not catching this earlier but don't we think that the override property should be named a little more general then "BlobFeed"? I expect us to use this for local builds as well so I wonder if this should just be "DotNetPackageOutputRootPath" or something similar.

Reply:
Well, I know it is in the correct order for Standard and CoreFX, at least. As long as Build.Common.props is imported before packaging msbuild data, then it will work... seems like a safe assumption, but I don't know if we have a generalized import order structure/convention to make that obvious.

About the name, currently "blob feed" [is defined](https://github.com/dotnet/source-build/tree/dev/release/2.0/Documentation/auto-dependency-flow#terms) as being local (disk) or remote (blob storage), so it is pretty general. I think it needs to be more specific than `DotNetPackageOutputRootPath` because it is *not* the directory where you should put your packages. [The property defines the root of the blob feed, and NuGet packages go in `<blob feed>/packages/`](https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/auto-dependency-flow/api.md#-pdotnetoutputblobfeeddirtarget-directory).

Reply:
Fair enough. This is mostly about the artifacts that will be taken from the output. We will also need to define some properties for bin and obj directories to share a common output but that is orthogonal to these.